python/python/lance/dataset.py

```diff
@@ -1679,16 +1691,30 @@ def create_index(
                     train_ivf_centroids_on_accelerator,
                 )
 
+                times = []
```

Comment:
nit: maybe use a dict and name the timers

Reply:
Done, but in the future we should probably just add a module for this